modules/ROOT/pages/manage-nodes.adoc

```diff
@@ -24,18 +24,25 @@ This script is included in the files you downloaded when installing Runtime Fabr
 +
 Ensure that you copy the environment file from the correct node type (controller or worker).
 
-. Modify the following environment variables in the environment file you just copied:
+. Modify the environment file you just copied to contain only the parameters required for the correct node type.
 +
-* RTF_PRIVATE_IP
-* RTF_DOCKER_DEVICE
-* RTF_ETCD_DEVICE (controller nodes only)
-* Remove RTF_ACTIVATION_DATA if it is present
-* RTF_INSTALL_ROLE=joiner
-* RTF_INSTALLER_IP=<IP address of the lead controller node>, as shown in the following example:
+The following example shows the environment parameters that are required for a worker node. Use the values in the file you copied for `RTF_DOCKER_DEVICE_SIZE`, `RTF_TOKEN`, and `RTF_INSTALLER_IP`. 
 +
-`RTF_INSTALLER_IP=10.1.255.93`
-* RTF_TOKEN
-* RTF_NODE_ROLE=worker_node (worker nodes only)
+** `RTF_NODE_ROLE=worker_node`
+** `RTF_INSTALL_ROLE=joiner`
+** `RTF_DOCKER_DEVICE_SIZE=250G`
+** `RTF_TOKEN="R6J4UuEkDTdPjYAO"`
+** `RTF_INSTALLER_IP="10.0.42.86"`
+** Remove RTF_ACTIVATION_DATA if it is present.
++
+The following example shows the environment parameters that are required for a dedicated node. If you copied the environment file from a controller node, delete all lines in the file except those shown in the example. Use the values in the file you copied for `RTF_DOCKER_DEVICE_SIZE` and `RTF_TOKEN`. For `RTF_INSTALLER_IP`, specify the IP address of the lead controller node.
++
+** `RTF_NODE_ROLE=dedicated_ingress_node`
+** `RTF_INSTALL_ROLE=joiner`
+** `RTF_DOCKER_DEVICE_SIZE=250G`
```

Comment:
Our template has this value default to 50G: https://github.com/mulesoft/rtf-install-scripts/blob/master/aws/inbound_traffic_controller_env.sh#L7